setup/bindir/cloud-setup-databases.in

```diff
@@ -188,7 +188,10 @@ for full help
         sys.exit(1)
 
     def setupDBSchema(self):
-        if not self.rootuser:
+        if self.options.schemaonly and self.rootuser != None:
+                    self.info("--schema-only and --deploy-as cannot be passed together\n", None)
```

Comment:
@Spaceman1984 It is better to check this, soon after parsing the options passed.

Reply:
Done